.jscsrc

```diff
@@ -0,0 +1,16 @@
+{
+  "maximumLineLength": 250,
+  "disallowDanglingUnderscores": { 
+  	"allExcept": [
+  		"_$http_", 
+  		"_formUrl", 
+  		"_generateListItems", 
+  		"_$compile_", 
+  		"_$rootScope_", 
+  		"_$controller_", 
+  		"_images_", 
+  		"_$q_", 
+  		"_$log_"
+  	] 
```

Comment:
ok, so i played  a little bit with this, when i was getting the error i googled the answer, which was to add those rules to my file (by then i didn't have the file), I created the file and added the rules, so no more errors. It turns out that if i dont have the file, or if the file is empty, those errors will show. 
But, as long as the file is present, and returns an object, the undescore rules are not needed